go/worker/compute/executor/committee/node.go

```diff
@@ -140,6 +143,9 @@ type Node struct { // nolint: maligned
 	schedulerMutex sync.RWMutex
 	scheduler      schedulingAPI.Scheduler
 
+	proposingTimeout     bool
+	proposingTimeoutLock sync.RWMutex
```

Comment:
Is a separate lock required or do all callers (except line 699) take the CrossNode lock anyway? In this case, just mention that this is guarded by the CrossNode lock (as below) and remove the methods?